taqueria-sdk/index.ts

```diff
@@ -146,9 +146,29 @@ const sanitizeArgs = (parsedArgs: ParsedArgs) : Promise<SanitizedArgs> =>
 
 const parseArgs = (unparsedArgs: Args): LikeAPromise<ParsedArgs, Failure<undefined>> => {
     if (unparsedArgs && Array.isArray(unparsedArgs) && unparsedArgs.length >= 2) {
+
```

Comment:
Mind moving the hacks to a separate function, such as `preprocessArgs` please? Isolates some of the noice that the hacks add to this function.